integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,21 @@
+{
+  "contentMetadata":
+  {
+    "name": "Web archive of @rt_com Twitter",
+    "mime": "application/wacz",
+    "description": "Web archive of @rt_com Twitter on 2022-03-29",
+    "createdBy": "Starling Lab",
+    "created": "2022-03-29T13:54:20Z",
+    "extras":
+    {
+      "authsignSoftware": "authsigner 0.3.1",
+      "authsignDomain": "authsign.stg.starlinglab.org",
+      "waczVersion": "1.1.1",
+      "pages":
+      {
+        "5ZwqT9xGkYVdYku3EVQLrD": "https://twitter.com/rt_com"
+      }
+    }
```

Comment:
@YurkoWasHere am I understanding this right based on your Slack msg?

```suggestion
    },
    "private":
    {
      "crawlConfig":
        {
          "crawl_id": "2653b8981076",
          "crawl_template_id": "0055c2ab-cf8c-458b-9321-0f931800c361",
          "crawl_config":
          {
            "name": "1646326337.674519",
            "colls": [],
            "crawlTimeout": 300,
            "scale": 1,
            "schedule": "",
            "runNow": false,
            "config":
            {
              "seeds": [
                "https://twitter.com/mihali_mow/status/1499185490762911744?s=21"
              ],
              "scopeType": "page",
              "depth": -1,
              "limit": 0,
              "extraHops": 0,
              "behaviorTimeout": 60,
              "behaviors": "autoscroll,autoplay,autofetch,siteSpecific"
            }
          },
          "additional_data":
          {
            "url_list": "['https://twitter.com/mihali_mow/status/1499185490762911744?s=21']",
            "thread_ts": "1646326337.674519",
            "ts": "1646326337.674519",
            "client_msg_id": "76da98c5-8a40-4f9d-af03-6478e962856f",
            "attached_service": "['twitter']",
            "attached_ts": "[1646268649]",
            "attached_author": "['Mihali']",
            "attached_text": "['А вот как теперь эти суки промывают мозги детям, чтобы выросло новое поколение долбо@бов…:imp:']",
            "country_codes": "['latvia flag']",
            "all_codes": "[]"
          }
        }
    }
```

Reply:
Good idea if we can do this. The stuff here has flexible keys. Shall the key be `crawlConfig` or something else?

Reply:
OOops just realized that what i posed was the CRAWL data not the Crawl Config data. Maybe we should include that as well

The API call is to the `crawlconfigs` collection. The ui calles it a crawl template

https://browsertrix.stg.starlinglab.org/api/docs#/crawlconfigs/get_crawl_configs_archives__aid__crawlconfigs_get

```

  "crawlConfigs": [
    {
      "id": "string",
      "schedule": "",
      "config": {
        "seeds": [
          "string",
          {
            "url": "string",
            "scopeType": "prefix",
            "include": "string",
            "exclude": "string",
            "sitemap": true,
            "allowHash": true,
            "depth": 0
          }
        ],
        "scopeType": "prefix",
        "include": "string",
        "exclude": "string",
        "depth": -1,
        "limit": 0,
        "extraHops": 0,
        "behaviorTimeout": 0,
        "workers": 0,
        "headless": true,
        "generateWACZ": true,
        "combineWARC": true,
        "logging": "string",
        "behaviors": "autoscroll,autoplay,autofetch,siteSpecific"
      },
      "name": "string",
      "created": "2022-04-13T16:47:53.981Z",
      "colls": [],
      "crawlTimeout": 0,
      "scale": 1,
      "aid": "string",
      "userid": "string",
      "crawlCount": 0,
      "lastCrawlId": "string",
      "lastCrawlTime": "2022-04-13T16:47:53.981Z",
      "lastCrawlState": "string",
      "newId": "string",
      "oldId": "string",
      "inactive": false,
      "currCrawlId": "string",
      "userName": "string"
    }
  ]
}
```

Reply:
Feel free to include both as `crawlConfigs` and `crawlData`.